src/deploy/functions/build.ts

```diff
@@ -494,10 +494,10 @@ export function toBackend(
         nullsafeVisitor((cpu) => (cpu === "gcf_gen1" ? cpu : r.resolveInt(cpu)))
       );
       if (bdEndpoint.vpc) {
+        bkEndpoint.vpc = { connector: params.resolveString(bdEndpoint.vpc.connector, paramValues) };
```

Comment:
You want to resolve the `bdEndpoint.vpc.connector` and save that back into `bdEndpoint` so the `if` check below works. Something like this should do it:
```
if (bdEndpoint.vpc) {
        **bdEndpoint.vpc.connector = params.resolveString(bdEndpoint.vpc.connector, paramValues);**
        if (bdEndpoint.vpc.connector && !bdEndpoint.vpc.connector.includes("/")) {
          bdEndpoint.vpc.connector = `projects/${bdEndpoint.project}/locations/${region}/connectors/${bdEndpoint.vpc.connector}`;
        }
        **bkEndpoint.vpc = { connector: bdEndpoint.vpc.connector };**
        proto.copyIfPresent(bkEndpoint.vpc, bdEndpoint.vpc, "egressSettings");
      } ...
```

Reply:
thanks!